### ISSUE TITLE
DFTemplate.from_code breaks if the template contains an item with a custom tag that has the value 'true'

### ISSUE THREAD

**Issue by ryanbuck182** [Amp63/pyre#4]
It seems to be trying to iterate over a boolean. I think this is because of this function for StyledString in style.py:

```
    @staticmethod
    def from_nbt(nbt: str):
        try:
            nbt_dict = json.loads(nbt) <-------- Line A
            if 'text' in nbt_dict: # <-------------- Line B
                return StyledString.from_nbt_dict(nbt_dict)
            raise McItemlibStyleException('String is not a formatted styled string.')
        except json.JSONDecodeError:
            raise McItemlibStyleException('Invalid JSON string.')
```

When nbt is string 'true', Line A sets nbt_dict to boolean True. Line B then tries to iterate over nbt_dict, which is, of course, a boolean, not an iterable value.

**Comment by 7Limes** [comment]
Should be fixed in 0.6.2, lmk if you find any other bugs!

**Comment by Amp63** [comment]
Should be fixed in 0.6.2, lmk if you find anything else!
